### ISSUE TITLE
Pip Install Doesn't have Six Dependency

### ISSUE THREAD

**Issue by elyssonmr** [matheuscas/pycpfcnpj#12]
Hello how are you guys?
I've installed this library and I notice that it requires *six* library. Looks like this lib doesn't have it within its dependencies.
As soon as I arrive home I'll make a pull request adding it.

Thanks

**Comment by matheuscas** [comment]
Hi, thanks for submitting an issue. But this is odd, requirements [has `six` as dependency](https://github.com/matheuscas/pycpfcnpj/blob/master/requirements.txt). Maybe another one related or dependent? Thanks again.

**Comment by elyezer** [comment]
Six is listed on the requirements file but when you install a lib, usually, you do that by running pip and the dependencies should also be listed on install_requires. For more information check [1]

I personally avoid requirements for the lib installation by doing this  [2] but have all the dependencies on the setup.py. I like to have a requirements-dev.txt with only the packages needed to develop the package though.

[1] https://packaging.python.org/discussions/install-requires-vs-requirements/
[2] https://github.com/SatelliteQE/betelgeuse/blob/master/requirements.txt

**Comment by matheuscas** [comment]
Hey @elyezer , I think this is a great solution to make dependencies lighter. Will you fill a PR about this? It would be awesome. One more time, thanks again.

**Comment by elyezer** [comment]
@matheuscas I think @elyssonmr will do that, let's see :smile:

**Comment by matheuscas** [comment]
As I said in @elyssonmr PR, I've closed it to give place to a recent PR that was doing the same and a bit more.  I've realized that the main idea of this small project was to have no dependency at all. I've been forget this in some recent PRs and I let it pass. But thanks to the latest PR of @elyssonmr and the other one from @ozeiasc, this came up again to my mind and I've made I small change to use `sys` rather than `six` to check if we are dealing with python 2 or 3. Check it out.
